docs/api-guide/serializers.md

```diff
@@ -382,7 +382,7 @@ For example, suppose we wanted to ensure that `User` instances and `Profile` ins
 This manager class now more nicely encapsulates that user instances and profile instances are always created at the same time. Our `.create()` method on the serializer class can now be re-written to use the new manager method.
 
     def create(self, validated_data):
-        return User.objects.create(
+        return UserManager.objects.create(
```

Comment:
did you test it manually?

Reply:
My bad. Didn't test it out before submitting PR.\
I thought we had to use the `UserManager` class now to create the objects and didn't realise that we needed to add `objects: UserManager()` in the `User` class itself to override the create method.

Reply:
exactly. thanks